File: SchoolManagement.Domain/SupplierInformation.cs
using SchoolManagement.Domain.Common;

namespace SchoolManagement.Domain
{
    public class SupplierInformation : BaseDomainEntity
    {
        public SupplierInformation()
        {
           

        }

        public int SupplierInformationId { get; set; }
        public int? BaseSchoolNameId { get; set; }
        public int? BookInformationId { get; set; }
        public string? SupplierName { get; set; }
        public string? Address { get; set; }
        public string? Phone { get; set; }
        public string? Email { get; set; }
        public DateTime? SuppliedDate { get; set; }
        public DateTime? ReceivedDate { get; set; }
        public string? Remarks { get; set; }
        public int? MenuPosition { get; set; }
        public bool IsActive { get; set; }

        public virtual BaseSchoolName? BaseSchoolName { get; set; }
        public virtual BookInformation? BookInformation { get; set; }


    }
}

File: ClientApp/src/app/book-management/book-management-tab/models/SupplierInformation.ts
export interface SupplierInformation {
    supplierInformationId: number;
    baseSchoolNameId: number;
    bookInformationId: number;
    supplierName: string;
    address: string;
    phone:string;
    email: string;
    suppliedDate: Date;
    receivedDate: Date;
    remarks: string;
    //menuPosition: string;
    isActive: boolean;
}